tests/historic_results/ArEntail.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Which model were these run with? Maybe it makes sense to put them in a folder with the model name?

Reply:
randomly selected from the results folder so multiple (I believe most is e5-mult-small).